### ISSUE TITLE
All default icons dissapeared in polyMC 1.4.x

### ISSUE THREAD

**Issue by Ember-Ember** [PolyMC/PolyMC#1009]
### Version of PolyMC

PolyMC 1.4.0-1.4.1 non-portable .tar.gz

### Description of bug

I Updated PolyMC from 1.3.0, all default instance icons and most launcher icons dissapeared

PolyMC 1.3.0
![Screenshot from 2022-08-01 23-07-30](https://user-images.githubusercontent.com/48266942/182156704-03f458cc-76b7-42f1-a215-fa33dce382d2.png)
![Screenshot from 2022-08-01 23-10-44](https://user-images.githubusercontent.com/48266942/182156813-dfad70bc-e7fb-45db-9daf-957538172e99.png)

PolyMC 1.4.1
![Screenshot from 2022-08-01 23-09-30](https://user-images.githubusercontent.com/48266942/182156930-5412ece3-4059-4622-be5b-6b50e017674f.png)
![Screenshot from 2022-08-01 23-09-36](https://user-images.githubusercontent.com/48266942/182156986-5959b541-bbc5-439b-aa6a-60b1602356d6.png)

### Steps to reproduce

Update PolyMC from 1.3.x (1.3.0 in my case), all default instance icons and most launcher icons are gone

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by DioEgizio** [comment]
I tried to investigate this, works fine on Ubuntu 20.04, so this must be something wrong with cinnamon

**Comment by DioEgizio** [comment]
Seems to work fine in Linux Mint 21, maybe update to that

**Comment by Ember-Ember** [comment]
> Seems to work fine in Linux Mint 21, maybe update to that

I'll check again when I'm able to upgrade

**Comment by Ember-Ember** [comment]
I can confirm that it's not an issue in mint 21 while it is in 20.3. 
however, i didn't test loading another icon style, turns out it just needs to be refreshed, though the icons do disappear again if it's closed. guess i just have to wait for 21 to be available to upgrade to :shrug:

**Comment by txtsd** [comment]
Do y'all have the equivalent of `qt6-svg` and `qt6-imageformats` installed?

**Comment by Ember-Ember** [comment]
> Do y'all have the equivalent of `qt6-svg` and `qt6-imageformats` installed?

does not appear so (seems only qt5 versions) nor do i know how i would install those

**Comment by txtsd** [comment]
Perhaps someone familiar with Ubuntu packages can help.

**Comment by DioEgizio** [comment]
qt5-svg and qt5-imageformats

**Comment by txtsd** [comment]
I think these icon errors are only happening on the Qt6 builds.

**Comment by DioEgizio** [comment]
What are you talking about, they aren't, Ubuntu 20.04 and derivates don't even have qt6...

**Comment by txtsd** [comment]
Derp

**Comment by DioEgizio** [comment]
Btw for now you can use the flatpak as a workaround, it'd work fine there

**Comment by JoeJoeTV** [comment]
Hey, I also have noticed this issue, since I'm still on Linux Mint Cinnamon 20.3.
I have experimented a bit and you can fix the issue by doing the following:

1. Go to the PolyMC settings and change the current icon theme to a different one and close the settings
2. Go to the settings AGAIN and select your desired icon theme again and close the settings, then the icons should be displayed correctly as far as I can tell.

This resets again after reopening PolyMC, so that's the problem and it looks like it's trying to pull some icons from some default theme if I'm correct.

**Comment by Ember-Ember** [comment]
> This resets again after reopening PolyMC, so that's the problem and it looks like it's trying to pull some icons from some default theme if I'm correct.

Yeah, for me I think it took the notes App icon for one of the icons, and the java and minecraft ones as shown in the images

**Comment by JoeJoeTV** [comment]
It may be using the icon names from the PolyMC themes, that it's using internally any is trying to get icons using these from the system theme, but that's just a guess.

**Comment by Ember-Ember** [comment]
definitely something with mint 20.3 as upgrading fixed it
